package.json

```diff
@@ -10,6 +10,7 @@
     "build": "turbo run build",
     "build-next": "turbo run build-next",
     "build-payload": "turbo run build-payload",
+    "build-ts": "turbo run build-ts",
```

Comment:
Shouldn't this be `build` i.e. in vpnmanager, build will call build-ts but in other apps, it'll just call build.